app.json

```diff
@@ -2,6 +2,11 @@
   "name": "node-starter",
   "description": "",
   "formation": {},
+  "env": {
+    "GOOGLE_ANALYTICS": {
```

Comment:
The heroku review app doesn't seem to pick up the env variables from the base app unless you have this

Reply:
ah yes - I was planning on not having the PR reviews have GA in them 🙃  - thoughts?

Reply:
Whoops! Good idea, I reverted my change.